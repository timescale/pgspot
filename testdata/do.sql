
DO LANGUAGE PLPGSQL $$ BEGIN SELECT unsafe_call2(); END; $$;
DO LANGUAGE PLPGSQL $$ BEGIN PERFORM unsafe_call3(); END; $$;

-- nested DO
DO LANGUAGE PLPGSQL $$
BEGIN
  PERFORM unsafe_call8();
  DO $i$
  BEGIN
    PERFORM unsafe_call10();
  END;
  $i$;
  SET search_path TO pg_catalog,pg_temp;
  DO $i$
  BEGIN
    PERFORM safe_call17();
    RESET search_path;
    PERFORM unsafe_call19();
  END;
  $i$;
END;
$$;

-- unknown language
DO LANGUAGE PLLUA $$
  print("Hello World")
$$;

