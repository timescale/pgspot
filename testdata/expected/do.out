PS016: Unqualified function call: unsafe_call2 at line 2
PS016: Unqualified function call: unsafe_call3 at line 3
PS016: Unqualified function call: unsafe_call8 at line 4
PS016: Unqualified function call: unsafe_call10 at line 4
PS016: Unqualified function call: unsafe_call19 at line 4
Unknown language: pllua

Errors: 0 Warnings: 5 Unknown: 1
